src/Components/Clicker.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import ReactDOM from "react-dom"
+
+class Clicker extends React.Component{
+        
+    state = {
+        counter: 0,
+        increase: 1,
+        cost: 10
+    }
+      
+    playAgain = () => {
+        window.location.reload(false)
+    }
+      
+    handleClick = () => {
+        let { counter, increase } = this.state
+      
+        if (counter >= 99) {
+            const youWin = (
+            <h1 id='won'>You Won!</h1>
+          )
+            const playAgain = (
+            <button onClick={this.playAgain}>Play again</button>
+          )
+            ReactDOM.render(youWin, document.getElementById('won'))
```

Comment:
You don't need to do all of this to show the div you've created. You can simply return the h1 element out of this function, or have a if statement where the div only shows if there is a winner state.